lib/utils/room_status_extension.dart

```diff
@@ -120,4 +86,38 @@ extension RoomStatusExtension on Room {
     }
     return lastReceipts.toList();
   }
+
+  String _getLocalizedStatusGroupChat(BuildContext context) {
+    final totalMembers =
+        (summary.mInvitedMemberCount ?? 0) + (summary.mJoinedMemberCount ?? 0);
+
+    return L10n.of(context)!.membersCount(totalMembers.toString());
+  }
+
+  String _getLocalizedStatusDirectChat(
+    CachedPresence? directChatPresence,
+    BuildContext context,
+  ) {
+    if (directChatPresence != null) {
+      if (directChatPresence.presence == PresenceType.online) {
+        return L10n.of(context)!.onlineStatus;
+      }
+      final lastActiveDateTime = directChatPresence.lastActiveTimestamp;
+      final currentDateTime = DateTime.now();
+      if (lastActiveDateTime != null) {
+        if (lastActiveDateTime.isLessThanOneHourAgo()) {
+          return L10n.of(context)!.onlineMinAgo(
+            currentDateTime.difference(lastActiveDateTime).inMinutes,
+          );
+        } else if (lastActiveDateTime.isLessThanTenHoursAgo()) {
+          final timeOffline = currentDateTime.difference(lastActiveDateTime);
+          return L10n.of(context)!.onlineHourAgo(
+            timeOffline.inHours,
+            timeOffline.inMinutes - (timeOffline.inHours * 60),
+          );
+        }
+      }
+    }
+    return L10n.of(context)!.offline;
```

Comment:
and if the presence > 10h -> offline too.

Reply:
already the case